packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -183,9 +186,9 @@ - (void)showCamera {
   }
   // Camera is not available on simulators
   if ([UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera] &&
-      [UIImagePickerController isCameraDeviceAvailable:_device]) {
+      [UIImagePickerController isCameraDeviceAvailable:[self getCameraDevice:_arguments]]) {
     _imagePickerController.sourceType = UIImagePickerControllerSourceTypeCamera;
-    _imagePickerController.cameraDevice = _device;
+    _imagePickerController.cameraDevice = [self getCameraDevice:_arguments];
```

Comment:
Please use a local variable instead of calling this twice.

Reply:
done.